code/game/objects/items/storage/basket.dm

```diff
@@ -0,0 +1,6 @@
+/obj/item/storage/basket
```

Comment:
can you at least give these backpack or duffel bag-level storage capacity (even if you can't wear them on your back)?